File: 1033. Moving Stones Until Consecutive/Solution.py
from typing import List


class Solution:
    def numMovesStones(self, a: int, b: int, c: int) -> List[int]:
        a, b, c = sorted([a, b, c])
        if c - a == 2:
            return [0, 0]
        return [1 if min(b - a, c - b) <= 2 else 2, c - a - 2]

File: 1033. Moving Stones Until Consecutive/Solution.java
import java.util.Arrays;

class Solution {
    public int[] numMovesStones(int a, int b, int c) {
        int[] stones = {a, b, c};
        Arrays.sort(stones);
        if (stones[2] - stones[0] == 2) {
            return new int[]{0, 0};
        }
        return new int[]{Math.min(stones[1] - stones[0], stones[2] - stones[1]) <= 2 ? 1 : 2, stones[2] - stones[0] - 2};
    }
}
